sirepo/pkcli/job_agent.py

```diff
@@ -651,13 +651,11 @@ async def _prepare_simulation(self):
     def _sbatch_script(self):
         i = self.msg.shifterImage
         s = o = ''
-        q = "debug" if self.msg.sbatchHours < 0.5 \
-            and self.msg.sbatchCores < 62 * 32 else "regular"
         if i:
 #TODO(robnagler) provide via sbatch driver
             o = f'''#SBATCH --image={i}
 #SBATCH --constraint=haswell
-#SBATCH --qos={q}
+#SBATCH --qos={self.msg.sbatchQueue}
```

Comment:
This needs to be validated. It's a user input. We can't just insert it. It should match one of the known queues. This can be denormalized in the job_agent.

sbatchHours and sbatchCores are semi-validated (float conversion and comparisons), but it's wonky. They need to be actually validated, too. This should happen in job_api. sbatchCores should be a positive int, and sbatchHours should be a positive float.

Reply:
https://github.com/radiasoft/sirepo/pull/2511/commits/fb55c2011646a1e6d9593bfb0c069e4c4223b959